Failing dependency: blevesearch's regexp tokenizer moved

Hi!

When I try to build a project that depends on your multibayes package I get following error:
```
${GOPATH}/github.com/lytics/multibayes/tokenize.go:10:2: cannot find package "github.com/blevesearch/bleve/analysis/tokenizers/regexp_tokenizer" in any of: ...
```

I think, the package was moved on blevesearch's site. Could you update your code to reflect the changed dependency?

Thanks in advance.

--- a/tokenize.go
+++ b/tokenize.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/blevesearch/bleve/analysis"
-	"github.com/blevesearch/bleve/analysis/tokenizers/regexp_tokenizer"
+	regexp_tokenizer "github.com/blevesearch/bleve/analysis/tokenizer/regexp"
 	"github.com/blevesearch/go-porterstemmer"
 )
 

